td-agent/Gemfile

```diff
@@ -70,6 +70,7 @@ gem "fluent-plugin-metrics-cmetrics", "0.1.1"
 gem "fluent-plugin-calyptia-monitoring", "0.1.3"
 gem "opensearch-ruby", "1.0.0"
 gem "fluent-plugin-opensearch", "1.0.0"
+gem "fluent-plugin-windows-exporter", "1.0.0"
```

Comment:
Tests failed on non-windows platforms.
Please add `platforms: windows_platforms` (see the below block).

Reply:
Sorry! fixed by d26444498585035f9bb80261da1802629241b1d8.